includes/class-omise-money.php

```diff
@@ -0,0 +1,56 @@
+<?php
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * @since 3.5
+ */
+class Omise_Money {
+	/**
+	 * @var array
+	 */
+	private static $subunit_multiplier = array(
+		'AUD' => 100,
+		'CAD' => 100,
+		'CHF' => 100,
+		'CNY' => 100,
+		'DKK' => 100,
+		'EUR' => 100,
+		'GBP' => 100,
+		'HKD' => 100,
+		'JPY' => 1,
+		'MYR' => 100,
+		'SGD' => 100,
+		'THB' => 100,
+		'USD' => 100
+	);
+
+	/**
+	 * @param  int|float $amount
+	 *
+	 * @return float
+	 */
+	public static function purify_amount( $amount ) {
```

Comment:
Does this need to be public?

Reply:
I'm not sure this function is even necessary, but if we really must have it, we need to correct the exception message as this function is perfectly happy accepting numeric strings. i.e. `purify_amount("123")` works just fine

Reply:
@jonrandy any idea for the exception message? `An amount has to be number`?